src/Powercord/migrations/20220822-move-themes-plugins.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const { join } = require('path');
+
+module.exports = () => {
+  const oldPlugs = join(__dirname, '..', 'plugins');
+  const newPlugs = join(__dirname, '..', '..', '..', 'plugins');
+  const oldThemes = join(__dirname, '..', 'themes');
+  const newThemes = join(__dirname, '..', '..', '..', 'themes');
+
+
+  // This function recursively moves all subdirectories and files from src to dest
+  let moveRecursiveSync = function(src, dest) {
+    const exists = fs.existsSync(src);
+    if (exists)  {
+      fs.readdirSync(src).forEach((name) => {
+        fs.renameSync(join(src, name), join(dest, name));
+      });
+    }
+  };
+
+  moveRecursiveSync(oldPlugs, newPlugs);
+  fs.rmdirSync(oldPlugs);
```

Comment:
Check if source exists before doing this. `Error: ENOENT: no such file or directory`